### PR TITLE
fix convert exception in mapToCameraCapturerConfiguration function

### PR DESCRIPTION
**Pull request by Macrow**
#218

### CODE REVIEW

File: android/src/main/java/io/agora/rtc/base/BeanCovertor.kt [hunk 1]
```diff
@@ -158,8 +158,8 @@ fun mapToLiveInjectStreamConfig(map: Map<*, *>): LiveInjectStreamConfig {
 
 fun mapToCameraCapturerConfiguration(map: Map<*, *>): CameraCapturerConfiguration {
     return CameraCapturerConfiguration(
-            intToCapturerOutputPreference(map["preference"] as Int),
-            intToCameraDirection(map["cameraDirection"] as Int)
+            intToCapturerOutputPreference((map["preference"] as Double).toInt()),
```
Comment (LichKing-2234) [inline android/src/main/java/io/agora/rtc/base/BeanCovertor.kt]: could you ps change to `as Number`?

File: android/src/main/java/io/agora/rtc/base/BeanCovertor.kt [hunk 2]
```diff
@@ -158,8 +158,8 @@ fun mapToLiveInjectStreamConfig(map: Map<*, *>): LiveInjectStreamConfig {
...
+            intToCameraDirection((map["cameraDirection"] as Double).toInt())
```
Comment (LichKing-2234) [inline android/src/main/java/io/agora/rtc/base/BeanCovertor.kt]: same as above

**Review by LichKing-2234 — CHANGES_REQUESTED** [review]
Thanks for your PR, it is my mistake, and I have added my review comment.

File: android/src/main/java/io/agora/rtc/base/BeanCovertor.kt (hunk 1 above)
Comment (Macrow) [reply android/src/main/java/io/agora/rtc/base/BeanCovertor.kt]: accomplished

**Comment by Macrow** [comment]
pr updated.

**Pull request merged**